features/specific/predicate_message.feature

```diff
@@ -0,0 +1,72 @@
+@boost
+Feature: Predicate Message Feature
+
+  This is just a simple feature meant to test boost's 
+  validation macros that also output custom message.
+  Problem was Cucumber-Cpp didn't capture those messages
+  and even didn't recognize failed asserations.
```

Comment:
The description should capture the behaviour, not why you did it (you should find that in the PR).